packages/icons/icons.json

```
@@ -2774,28 +2774,28 @@
     {
         "displayName": "Bank account",
         "iconName": "bank-account",
-        "tags": "money, finance, funds",
+        "tags": "money, finance, funds, piggy bank",
         "group": "miscellaneous",
         "codepoint": 61745
     },
     {
         "displayName": "Cell tower",
         "iconName": "cell-tower",
-        "tags": "signal, communication, radio, mast",
+        "tags": "signal, communication, radio, mast, connection, aerial, radar, connectivity, cancer, internet",
```

how does "cancer" relate to this icon?

```suggestion
        "tags": "signal, communication, radio, mast, connection, aerial, radar, connectivity, internet, data, transmission, broadcast",
```

---

I think they were thinking radiation --> cancer. we can remove